packages/owl-bot/src/cmd.ts

```diff
@@ -25,7 +25,9 @@ export function newCmd(logger = console): Cmd {
     options?: proc.ExecSyncOptions | undefined
   ): Buffer => {
     logger.info(command);
-    return proc.execSync(command, options);
+    const output = proc.execSync(command, options);
+    if (typeof output === 'string') return Buffer.from(output);
```

Comment:
There was a breaking change in `@types/node` that bit us, this just corrects the type information.